backend/test_chat_fix.py

```
@@ -0,0 +1,108 @@
+                        print("The chat-with-progress endpoint is working correctly with planning-first architecture.")
+                        return True
+                    else:
+                        print("\n⚠️  No progress updates or final response received")
+                        return False
+                        
+                else:
+                    print(f"❌ Request failed with status: {response.status}")
+                    try:
+                        error_text = await response.text()
+                        print(f"   Error: {error_text}")
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)